webclient/components/def-processors/view/content-during-appt-of-a-single-ct.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <el-collapse>
+  <el-collapse @change="mfRowSlideToCurrent">
```

Comment:
Rec has 1 slide and not 6 slides if there are 6 rows.

Give better name in future

vueHorizontalListSetCurrentSlide